### PR TITLE
Posts section

### PR DESCRIPTION
**Pull request by vaidas33**

### CODE REVIEW

**Comment by anavisnia** [comment]
Screenshot_56
Truksta dizaino is css puses

File: css/home-page.css [hunk 1]
```diff
@@ -33,5 +33,33 @@ CUSTOM HOME PAGE STYLES GOES BELOW THIS COMMENT
...
+h3 {
+    text-transform: capitalize;
+    color: #000;
+    font-size: 33px;
+    padding-bottom: 30px;
+}
+
+h4 {
+    padding: 15px 0;
+}
```
Comment (anavisnia) [inline css/home-page.css:50]: Css style to add a border

**Review by anavisnia — DISMISSED** [review]
Looks good

**Review by anavisnia — APPROVED** [review]
LG

**Review by Alekslov — APPROVED** [review]

**Pull request merged**